src/platform/testing/e2e/test-server.js

```diff
@@ -2,15 +2,15 @@
 
 // Simple test server to serve up the build files.
 
-// This is used over a bear http-server invocation because it handles paths inside React apps
-// using the expression-history-api-fallback option.
+// This is used over a vanilla http-server because it handles paths
+// inside React apps using the express-history-api-fallback option.
 
-const fs = require('fs');
 const commandLineArgs = require('command-line-args');
 const express = require('express');
 const fallback = require('express-history-api-fallback');
-const path = require('path');
 const morgan = require('morgan');
+const path = require('path');
```

Comment:
Was the `path` import moved to be in alphabetical order?

Reply:
Yes.